main/options/options.h

```diff
@@ -151,7 +151,6 @@ struct Options {
     std::vector<std::string> dslRubyExtraArgs;
     std::string storeState = "";
     bool enableCounters = false;
-    std::vector<std::string> someCounters;
```

Comment:
We declared this but never wrote to it anywhere. There might have at one point been a flag that populated this, but there is no such flag anymore.